wgsl/index.bs

```diff
@@ -11734,18 +11735,31 @@ See [[#builtin-inputs-outputs]] for how to specify that a pipeline input or outp
       <td>output
       <td>vec4&lt;f32&gt;
       <td style="width:50%">Output position of the current vertex, using homogeneous coordinates.
-      After homogeneous normalization (where each of the *x*, *y*, and *z* components
-      are divided by the *w* component), the position is in the WebGPU normalized device
-      coordinate space.
+
+      Let the result be the vector (*x*,*y*,*z*,*w*).
+      Then *w* is known as the <dfn noexport>homogeneous divisor</dfn>, and
+      the position of the vertex in WebGPU [=normalized device coordinates=]
+      will be (*x*/*w*, *y*/*w*, *z*/*w*).
+
       See [[WebGPU#coordinate-systems]].
 
   <tr>
       <td>fragment
       <td>input
       <td>vec4&lt;f32&gt;
-      <td style="width:50%">Framebuffer position of the current fragment in [=framebuffer=] space.
-      (The *x*, *y*, and *z* components have already been scaled such that *w* is now 1.)
-      See [[WebGPU#coordinate-systems]].
+      <td style="width:50%">
+      The *x*, *y*, and *z* components are the interpolated position of the current fragment,
+      in viewport coordinates:
+      *  The *x* and *y* components are the position of the current fragment in the [=framebuffer=].
+          Each pixel has an extent of 1.0 unit in each of the x and y dimensions, and pixel centers
+          are at (0.5,0.5) offset from integer coordinates.
+      * The *z* component is the fragment depth, interpolated between the viewport minimum and maximum depths.
```

Comment:
The API spec has a an editorial TODO, essentially in the 'rasterize' abstract op, to describe this.

I based this off Vulkan and OpenGL, which are pretty clear about what's going on here.

Reply:
Editors' plan of record is to someday hopefully rewrite this section by reference to the Vulkan spec.